README.md

```diff
@@ -22,27 +22,17 @@ Java 8 or later is required to use the January 2020 client libraries, otherwise
 
 Each service might have a number of libraries available from each of the following categories:
 
-- [Client: GA January 2020 Releases](#Client-GA-January-2020-Releases)
+- [Client: New Releases](#Client-New-Releases)
```

Comment:
Update the following text as well in `Important` section since the reference to January 2020 GA libraries is removed.

```
The Azure SDK team is pleased to make available the January 2020 client library General Availability (GA) release.
```

Reply:
The anchor on line 68 will need to be fixed as well.  Please grep the file for "January" There's ~4 spots that will need to be fixed at the same time.